### ISSUE TITLE
Fixed Perspective of Point Cloud

### ISSUE THREAD

**Issue by GitwellAnyohub** [daavoo/pyntcloud#225]
Hi. I've been using this library to visualize my point clouds and I think it is awesome. I work with a lot of video data so up until now I've only been able to visualize one point cloud at a time. I'd like to paste together the point clouds from a bunch of frames to create a moving point cloud. The problem is anytime I do cloud.plot and I'm interactively look at my point cloud, I need to manually zoom in a little bit and translate the image in order to center to the scene of interest. Are there any parameters or things I can do to ensure that my plot is in the exact position and orientation that I want it (before any type of interacting). Another issue will be saving these as pngs so I can eventually paste them together into a video, but one thing at a time.

**Comment by TheDelus** [comment]
Hi, plotting multiple point clouds is something that would be of value for me as well. @GitwellAnyohub @daavoo Is someone already working on that? If not I could give it a go.

**Comment by daavoo** [comment]
Um. I think that there are many different things requestes. I will try to anwer / implement some soon

**Comment by banesullivan** [comment]
With #248 coming soon, you could use PyVista for rendering more than one dataset in a scene as well as updating the coordinates and saving a frame.

Maybe check out a few of PyVista's video/gif examples:

- [Create a GIF](https://docs.pyvista.org/examples/02-plot/plot-gif.html)
- [Create a Movie](https://docs.pyvista.org/examples/02-plot/plot-movie.html)

Simple code to add more than one PyntCloud to a scene (after the changes in #248):

```py
from pyntcloud import PyntCloud
import pyvista as pv

pc1 = PyntCloud.from_file("file1.ply")
pc2 = PyntCloud.from_file("file2.ply")

plotter = pv.Plotter()
plotter.add_mesh(pc1.to_pyvista(), color='red')
plotter.add_mesh(pc1.to_pyvista(), color='blue')
plotter.show()
```

Feel free to check out the rest of [PyVista's Examples](https://docs.pyvista.org/examples/index.html) and ping me/use [PyVista's support forum](https://github.com/pyvista/pyvista-support) if you have further question

**Comment by daavoo** [comment]
Closed in #248
